cmd/manager/main.go

```diff
@@ -440,6 +441,7 @@ func startOperator(stopChan <-chan struct{}) error {
 	}
 	params := operator.Parameters{
 		Dialer:            dialer,
+		IPFamily:          net.ToIPFamily(os.Getenv(settings.EnvPodIP)),
```

Comment:
I'm wondering if we should include that as an operator config setting, that defaults to autodetect if empty.
eg. `--ip-family=ipv4|ipv6|""`
So users can override it if for some reasons the IP family of the operator is not the ip family they want to use for the managed workload.

Reply:
I thought this would be a future addition. But I think you are right it would be good to have an escape hatch right from the start. Will add it.